app/react/socket.js

```diff
@@ -1,17 +1,17 @@
 import io from 'socket.io-client';
 import { isClient } from 'app/utils';
 
-let socket = { on: () => {} };
+let socket = { on: () => {}, off: () => {} };
+
 if (isClient) {
   //only websockets used, this allows for non sticky sessions on load balancer
   socket = io({ transports: ['websocket'], upgrade: false });
 }
 
-export default socket;
-
 const reconnectSocket = () => {
   socket.disconnect();
   socket.connect();
 };
 
+export default socket;
```

Comment:
Try to avoid use export default.  I know this was here before... is it too hard to change it?  Maybe not worth it right now. Thanks for putting the export at the bottom of the file.